### PR TITLE
Structural Refactor

### PR DESCRIPTION
**Pull request by mikesizz**
This PR is intended to clean up the API file hierarchy as well as bringing some small QoL improvements.

- Move all route handlers to their own files.
- Remove unused code.
- Add nodemon support for local development.
- Add prettier support.
- Group utility methods together.
- Minor Fixes
  - Correct express support for Promises.
  - Minor performance improvements - unnecessary iterations, Promise.all, etc.

### CODE REVIEW

**Comment by mikesizz** [comment]
This should not be merged until a functional test suite can be put in place (WIP on my local). This is intended to be a draft PR for review and discussion. TBH I am not sure if this is needed, or even desired, so any feedback before continuing would be appreciated. Just trying to help out.

Additional changes that should be included now or in the near future:
- functional test suite
- move configuration values to config.js with process.env overrides
  - current values as defaults
- GitHub PR template
- CHANGELOG.md
- Clean up .env file usage
  - Add .env.sample for ppl to copy
- License file
- Linter support
- Remove lambda configs if no longer needed

File: src/lib/conseil.js [hunk 1]
```diff
@@ -0,0 +1,636 @@
+const conseiljs = require('conseiljs')
```
Comment (mikesizz) [inline src/lib/conseil.js:1]: Nothing changed here.

**Comment by veqtor** [comment]
Hey, this is kind of stale, I can leave it as is now but likely we will change stuff around

**Comment by mikesizz** [comment]
@veqtor The idea was to build on top of this to avoid further utilization of the current code base. Nobody seems too interested in it, so we should go ahead and close it out.

**Comment by veqtor** [comment]
ok, but the idea was good though, since we need to modularize things more, sorry this got forgotten

**Pull request closed without merging**